File: src/packages/sdk/src/cli/profiles.ts
/**
 * Gate profile presets: strict, balanced, fast.
 * Profiles override flags unless explicitly set.
 */

export const PROFILES = {
  strict: {
    minScore: 95,
    maxDrop: 0,
    minN: 30,
    allowWeakEvidence: false,
  },
  balanced: {
    minScore: 90,
    maxDrop: 2,
    minN: 10,
    allowWeakEvidence: false,
  },
  fast: {
    minScore: 85,
    maxDrop: 5,
    minN: 5,
    allowWeakEvidence: true,
  },
} as const;

export type ProfileName = keyof typeof PROFILES;

File: src/packages/sdk-python/src/evalgate_sdk/cli/profiles.py
"""Gate profile presets.

Extracted to avoid typer dependency in config.py imports.
"""

from __future__ import annotations

PROFILES = {
    "strict": {"min_score": 95, "max_drop": 0, "warn_drop": 0, "min_n": 30, "allow_weak_evidence": False},
    "balanced": {"min_score": 90, "max_drop": 2, "warn_drop": 1, "min_n": 10, "allow_weak_evidence": False},
    "fast": {"min_score": 85, "max_drop": 5, "warn_drop": 2, "min_n": 5, "allow_weak_evidence": True},
}
